packages/message-list/src/vaadin-message-list-mixin.js

```diff
@@ -160,6 +165,14 @@ export const MessageListMixin = (superClass) =>
       }
     }
 
+    /** @private */
+    __debounceAppending() {
```

Comment:
This effectively enables scroll snapping and then turns it off again automatically after the decouncer triggers. So maybe `__enableScrollSnapping` would be a clearer name, plus a comment above the debouncer that it turns it off again after there are no further changes to items?

Reply:
Renamed, and added the comment